sema4ai/vscode-client/src/activities.ts

```diff
@@ -828,31 +836,22 @@ export async function createRobot() {
     }
 }
 
-export async function createTaskOrActionPackage() {
+export async function createPackage() {
     const TASK_PACKAGE = "Task Package (Robot)";
     const ACTION_PACKAGE = "Action Package";
-
-    const sidebarAgentPackageOrganizationSelection = getSelectedAgentPackageOrganization();
```

Comment:
Note: removed this feature for now as some additional work is needed for it (so, it'll always ask for the agent even if it's already selected in the tree).